express-get-json/index.js

```diff
@@ -0,0 +1,35 @@
+var express = require('express');
+var app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  const newGradesArr = [];
+  for (const key in grades) {
+    newGradesArr.push(key, grades[key]);
```

Comment:
This is pushing string keys and objects into the array try removing the `key` argument.

```js
newGradesArr.push(grades[key]);
```